test/integration/esm.spec.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+var run = require('./helpers').runMochaJSON;
+var utils = require('../../lib/utils');
+
+if (!utils.supportsEsModules()) return;
+
+describe('esm', function() {
+  it('should pass a passing esm test that uses esm', function(done) {
+    run(
+      'esm/esm-success.fixture.mjs',
+      ['--experimental-modules', '--no-warnings'],
```

Comment:
should be an empty array when Node >=13.2.0

Reply:
True! Done.